Sources/Controllers/2.0/Settings/ImportExport/OAExportItemsViewController.mm

```diff
@@ -96,11 +136,66 @@ - (long)getItemSize:(NSString *)item
     return attrs.fileSize;
 }
 
-- (IBAction) primaryButtonPressed:(id)sender
+- (void)updateFileSize
+{
+    _itemsSize = [self calculateItemsSize:self.getSelectedItems];
+    NSString *byteCount = [NSByteCountFormatter stringFromByteCount:_itemsSize countStyle:NSByteCountFormatterCountStyleFile];
+    NSString *approximateFileSize = [NSString stringWithFormat:OALocalizedString(@"approximate_file_size"), byteCount];
+    _fileSize = approximateFileSize;
+}
+
+- (UIView *)getHeaderForTableView:(UITableView *)tableView withFirstSectionText:(NSString *)text boldFragment:(NSString *)boldFragment forSection:(NSInteger)section
```

Comment:
no, this is an extended version from OABaseSettingsWithBottomButtonsViewController

Reply:
This method should not exist in the OABaseSettingsWithBottomButtonsViewController either. Please extract it to OAUtilities and prefer extending the existing functionality with additional parameters rather than copy/pasting and adding few lines.